src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -78,6 +78,12 @@ void NotationViewInputController::init()
             setViewMode(ViewMode::PAGE);
         });
 
+        if (globalConfiguration()->devModeEnabled()) {
+            dispatcher()->reg(this, "view-mode-float", [this]() {
```

Comment:
Doesn't this here take care of showing that float mode only in dev mode?

Reply:
This is just registering a handler, what to do when such an action is sent.
I admit that there may be a check for registered actions, but it’s unlikely...

Reply:
What would you propose to to do to prevent that option to be shown outside dev mode?

Reply:
We can affect the list of actions (`ALL_MODE_MAP`) or the method of generating list elements for the user (`NotationStatusBarModel::makeAvailableViewModeList` method)